### DISCUSSION TITLE
posts/meet-file-browser/

### DISCUSSION THREAD

**Discussion by giscus** [techno-tim/techno-tim.github.io#314] [Comments]
# posts/meet-file-browser/

Meet File Browser, an open source, self-hosted alternative to services like Dropbox and other web based file browsers.Today we’ll configure a containerized version of File Browser and have you up and going in just a few minutes.We’ll also walk through creating, editing, moving, copying, and even sharing files and folders so that you get a better understanding about what File Browser is all about.

https://technotim.live/posts/meet-file-browser/

**Comment by CRK1918** [comment]
look like linuxserver.io is no longer supported? I create with unhealthy docker status.
